crates/turborepo-lib/src/run/task_id.rs

```diff
@@ -20,8 +20,8 @@ pub struct TaskId<'a> {
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize, Hash)]
 #[serde(try_from = "String", into = "String")]
 pub struct TaskName<'a> {
-    package: Option<Cow<'a, str>>,
-    task: Cow<'a, str>,
+    pub(crate) package: Option<Cow<'a, str>>,
+    pub(crate) task: Cow<'a, str>,
```

Comment:
As my comment above displays, I really think we should keep these private and force callers to use the parsing logic in `From<&str> for TaskName` 

```suggestion
    package: Option<Cow<'a, str>>,
    task: Cow<'a, str>,
```